fili-core/src/main/java/com/yahoo/bard/webservice/logging/RequestLog.java

```diff
@@ -86,54 +86,6 @@ private RequestLog(RequestLog rl) {
     }
 
     /**
-     * Represents a phase that is timed.
-     * TimedPhase is used to associate a Timer located in the registry with the exact duration of such a phase for a
-     * specific request.
-     */
-    private static class TimedPhase {
-        private final String name;
-        private long start;
-        private long duration;
-
-        /**
-         * Constructor.
-         *
-         * @param name  Name of the phase
-         */
-        private TimedPhase(String name) {
-            this.name = name;
-        }
-
-        /**
-         * Start the phase.
-         */
-        private void start() {
-            if (start != 0) {
-                LOG.warn("Tried to start timer that is already running: {}", name);
-                return;
-            }
-            start = System.nanoTime();
-        }
-
-        /**
-         * Stop the phase.
-         */
-        private void stop() {
-            if (start == 0) {
-                LOG.warn("Tried to stop timer that has not been started: {}", name);
-                return;
-            }
-            duration += System.nanoTime() - start;
-            REGISTRY.timer(name).update(duration, TimeUnit.NANOSECONDS);
```

Comment:
I think this is getting lost. It may very well be that `TimedPhase::stop` needs to do the registration that `TimedPhase::close` is doing? Otherwise, I think there's a gap where things could be lost.

Or, to put it a different way, `stop` did the registering before, and now only `close` does it. Since `stop` was private, I don't know how much impact it will really have, but it's worth noting.